src/lib/math.js

```diff
@@ -18,8 +18,41 @@ const $builtinmodule = function (name) {
         return new Sk.builtin.float_(Math.ceil(_x));
     };
 
-    function comb(x, y) {
-        throw new Sk.builtin.NotImplementedError("math.comb() is not yet implemented in Skulpt");
+    function comb(n, k) {
+        n_js = Sk.misceval.asIndexOrThrow(n);
+        k_js = Sk.misceval.asIndexOrThrow(k);
```

Comment:
`let`

Reply:
Oh man, is it possible to automatically run eslint over these modules as part of the build process?

Reply:
You can probably add an eslint plugin to your editor?